Firestore/core/src/firebase/firestore/local/query_data.h

```diff
@@ -101,20 +102,20 @@ class QueryData {
     return snapshot_version_;
   }
 
-  const std::vector<uint8_t>& resume_token() const {
+  const nanopb::ByteString& resume_token() const {
     return resume_token_;
   }
 
   QueryData Copy(model::SnapshotVersion&& snapshot_version,
-                 std::vector<uint8_t>&& resume_token) const;
+                 nanopb::ByteString&& resume_token) const;
 
  private:
-  const core::Query query_;
+  core::Query query_;
   model::TargetId target_id_;
   model::ListenSequenceNumber sequence_number_;
   QueryPurpose purpose_;
-  const model::SnapshotVersion snapshot_version_;
-  const std::vector<uint8_t> resume_token_;
+  model::SnapshotVersion snapshot_version_;
+  nanopb::ByteString resume_token_;
```

Comment:
I understand that `nanopb::ByteString` comes in handy here, but I'm not sure that:
* `nanopb` types should really propagate outside of `Serializer`;
* the malloc-created pointer to a nanopb-type is really the best representation for a resume token.